src/main/java/interface_adapter/search/SearchController.java

```diff
@@ -0,0 +1,53 @@
+package interface_adapter.search;
+
+import data_access.movies.MovieDBDataAccessException;
+import use_case.search.SearchByCriteriaInputData;
+import use_case.search.SearchByCriteriaOutputData;
+import use_case.search.SearchInputBoundary;
+import use_case.search.SearchOutputBoundary;
+
+import java.util.Collections;
+
```

Comment:
```suggestion

import java.util.Collections;

import data_access.movies.MovieDBDataAccessException;
import use_case.search.SearchByCriteriaInputData;
import use_case.search.SearchByCriteriaOutputData;
import use_case.search.SearchInputBoundary;
import use_case.search.SearchOutputBoundary;

```
Checkstyle wants these reorganized